api/proquest/importer.py

```diff
@@ -390,6 +390,17 @@ def _get_book(self, patron, configuration, document_id):
 
             iterations += 1
 
+    def _parse_identifier(self, identifier):
+        """Parse the identifier and return an Identifier object representing it.
+
+        :param identifier: String containing the identifier
+        :type identifier: str
+
+        :return: Identifier object
+        :rtype: Identifier
+        """
+        return parse_identifier(self._db, identifier)
```

Comment:
Is this still being used? Since it's a private method I think you can remove it after changing to `parse_identifier`.

Reply:
Never mind, you use it in the core branch.